src/Main.java

```diff
@@ -1,278 +1 @@
-package com.company;
 
-import javax.swing.text.StyledEditorKit;
-import java.util.Scanner;
-import java.io.File;
-import java.io.*;
-import java.util.Queue;
-import java.util.*;
-
-class Cell{
-    int x;
-    int y;
-    int gCost;
```

Comment:
What is gCost? Space is cheap, you should consider using variable names that make sense to others.